tests/ui/offset-of/offset-of-enum.rs

```diff
@@ -9,5 +9,9 @@ enum Alpha {
 
 fn main() {
     offset_of!(Alpha::One, 0); //~ ERROR expected type, found variant `Alpha::One`
-    offset_of!(Alpha, Two.0); //~ ERROR no field `Two` on type `Alpha`
+    offset_of!(Alpha, One); //~ ERROR `One` is an enum variant; expected field at end of `offset_of`
```

Comment:
Nit: might be a good idea to include a note about how to write `One.0`. Users don't always remember that tuple struct/enum fields are named that way. 

This can absolutely be deferred to a future PR 🙂

Reply:
I would say that this feels like scope creep for this PR, particularly considering that the syntax isn't finalized, so I'll leave this as-is.